tests/e2e/test_rosacli_verify.go

```diff
@@ -199,13 +198,14 @@ var _ = Describe("Verify",
 				}
 
 			})
+
 		It("bring your own kms key functionality works on cluster creation - [id:60082]",
 			labels.Day1Post,
 			labels.Critical,
+			labels.NonHCPCluster,
 			func() {
 				By("Confirm current cluster profile uses kms keys")
-				curProfile := profilehandler.LoadProfileYamlFileByENV()
-				if !curProfile.ClusterConfig.KMSKey {
+				if clusterConfig.Encryption == nil {
```

Comment:
There are two fields for clusterConfig.Encryption struct.I think you can use clusterConfig.EnableCustomerManagedKey  to confirm

Reply:
Not present in classic cluster